### PR TITLE
Doc: Document the X-Grafana-Org-Id HTTP header

### LINKED ISSUE #26769: Document the X-Grafana-Org-Id HTTP header
Document the X-Grafana-Org-Id HTTP header in the API docs.

Added by PR https://github.com/grafana/grafana/pull/8122
Referenced here: https://community.grafana.com/t/organization-state-in-the-api/7954

### PR DESCRIPTION
**Pull request by ying-jeanne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #26769

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ying-jeanne** [comment]
the functionality tested

File: docs/sources/http_api/auth.md [hunk 1]
```diff
@@ -106,6 +107,8 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
   "secondsToLive": 86400
 }
 ```
+Header
+- **X-Grafana-Org-Id** - The org to which the created key belongs. It is optional. If it is not set, the created key would belongs to the current context org.
```
Comment (achatterjee-grafana) [inline docs/sources/http_api/auth.md:111]:
```suggestion
- **X-Grafana-Org-Id** - An optional property that specifies the org to which the created key belongs. If it is not set, the created key belongs to the current context org.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Thanks for adding this information! One minor change requested.

**Comment by ying-jeanne** [comment]
https://grafana.com/docs/grafana/latest/http_api/auth/ in this page, add a separate subsection as Token.

File: docs/sources/http_api/auth.md [hunk 2]
```diff
@@ -11,6 +11,27 @@ aliases = ["/docs/grafana/latest/http_api/authentication/"]
 
 Currently you can authenticate via an `API Token` or via a `Session cookie` (acquired using regular login or OAuth).
 
+## Header
```
Comment (marefr) [inline docs/sources/http_api/auth.md:14]:
```suggestion
## X-Grafana-Org-Id Header
```

**Review by marefr — APPROVED** [review]
LGTM, but added a suggestion. Lets see if @achatterjee-grafana agrees

File: docs/sources/http_api/auth.md [hunk 3]
```diff
@@ -11,6 +11,27 @@ aliases = ["/docs/grafana/latest/http_api/authentication/"]
...
+
+**X-Grafana-Org-Id** is an optional property that specifies the org to which the action is applied on. If it is not set, the created key belongs to the current context org. You could use this header in all requests excepts requests regarding admin.
```
Comment (achatterjee-grafana) [inline docs/sources/http_api/auth.md:16]:
```suggestion
**X-Grafana-Org-Id**  is an optional property that specifies the organization to which the action is applied. If it is not set, the created key belongs to the current context org. Use this header in all requests except those regarding admin.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Cleaned up the doc content.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM!

**Comment by marefr** [comment]
@achatterjee-grafana @ying-jeanne I suppose we should backport this to v7.5.x if we want these docs to show up under the latest tag?

**Comment by achatterjee-grafana** [comment]
Will do.

**Comment by achatterjee-grafana** [comment]
> @achatterjee-grafana @ying-jeanne I suppose we should backport this to v7.5.x if we want these docs to show up under the latest tag?

Done!

**Comment by marefr** [comment]
@achatterjee-grafana looks like you manually backported this? You could just have added the label "backport v7.5.x" and grafanabot would have opened the backport pr for you.

**Comment by achatterjee-grafana** [comment]
We spoke about the backport after the merge. The doc team is using that label in other PRs.

**Pull request merged**